container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/user_services_functions/start_user_services.go

```diff
@@ -552,6 +556,18 @@ func createStartServiceOperation(
 			envVars[key] = strings.Replace(envVars[key], privateIPAddrPlaceholder, privateIPAddrStr, unlimitedReplacements)
 		}
 
+		// THIS IS A HACK
```

Comment:
i like that the hack is isolated to one function and compose transpilation this is good